selfdrive/car/honda/carstate.py

```diff
@@ -255,6 +255,8 @@ def update(self, cp, cp_cam, cp_body):
 
     if self.CP.carFingerprint in HONDA_BOSCH_ALT_BRAKE_SIGNAL:
       ret.brakePressed = cp.vl["BRAKE_MODULE"]["BRAKE_PRESSED"] != 0
+    elif self.CP.carFingerprint == CAR.HRV_2023:
+      ret.brakePressed = cp.vl["POWERTRAIN_DATA"]["BRAKE_PRESSED"] != 0
```

Comment:
As mentioned in Discord, the `BRAKE_SWITCH` signal was added to prevent a mismatch with the PCM and openpilot, where a tiny amount of braking force would be seen as braking by the car, but not openpilot, causing a fault. Before we can remove it, we need to make sure that the car does not use it for anything.

Reply:
@AlexandreSato I reverted this since it was causing controls mismatches.